Functions.Templates/Templates/DurableFunctionsOrchestration-CSharp-Isolated/function.json

```diff
@@ -0,0 +1,9 @@
+{
+  "bindings": [
+    {
+      "name": "context",
+      "type": "orchestrationTrigger",
```

Comment:
What is this file for?

Reply:
You don't need this for this template. You only need this for .csx templates.